code/datums/datum.dm

```diff
@@ -183,13 +183,20 @@
 	to_chat(target, txt_changed_vars())
 #endif
 
-///Return a LIST for serialize_datum to encode! Not the actual json!
-/datum/proc/serialize_list(list/options)
-	CRASH("Attempted to serialize datum [src] of type [type] without serialize_list being implemented!")
+/// Return a list of data which can be used to investigate the datum, also ensure that you set the semver in the options list
+/datum/proc/serialize_list(list/options, list/semvers)
+	SHOULD_CALL_PARENT(TRUE)
+
+	. = list()
+	.["tag"] = tag
 
-///Accepts a LIST from deserialize_datum. Should return src or another datum.
+	SET_SERIALIZATION_SEMVER(semvers, "1.0.0")
```

Comment:
Should this version not be a define or something? I'm not 100% as to what the versioning here stands for, mind you.

Reply:
versions are not going to be a define